.github/workflows/lint.yml

```diff
@@ -52,11 +52,20 @@ env:
   # the subset of development dependencies installed.
   INSTALL_EXTRA: lint
 
+  ORG_REPO: trailofbits.com/.github
+  ORG_REPO_REF: ${{ github.job_workflow_sha }}
+  ORG_REPO_PATH: /tmp/org_github
```

Comment:
Wait, does this work? 🤯 

For some reason I had trouble getting the `job_workflow_sha` to evaluate to anything nonempty, but maybe I was using it in the wrong context.

Reply:
not sure yet, its still failing on my env syntax on ORG_REPO 😄

Reply:
looks like no, this doesn't work. I think it'll be fine to just use main

Reply:
Yeah, that works for me (I wanted to avoid it because it isn't pinned, but IMO that's relatively minor since we don't expect to change this file much.)

Reply:
d'oh I had the org repo wrong. I tried it again after fixing that and it does work, but its the head ref of main for some reason 🤔

Reply:
I just made it a configurable input so at least we have that. Other than that this seems to be working 🎉

Reply:
Nice!